data-prepper-core/src/main/java/org/opensearch/dataprepper/plugin/PluginConfigurationConverter.java

```diff
@@ -31,6 +31,10 @@ class PluginConfigurationConverter {
     PluginConfigurationConverter(final Validator validator,
                                  @Named("pluginConfigObjectMapper")
                                  final ObjectMapper objectMapper) {
+        final SimpleModule simpleModule = new SimpleModule();
```

Comment:
@Durdush , Thank you for this change!

It looks like the code has changed some since the original issue. This should be configured here instead:

https://github.com/opensearch-project/data-prepper/blob/a407e4f9eef2a82f4e1b3c078e9195463063fe1b/data-prepper-core/src/main/java/org/opensearch/dataprepper/plugin/ObjectMapperConfiguration.java#L19-L20

Also, it would be good to have a unit test which verifies that the ObjectMapper can deserialize Duration and ByteCount objects.